doc/docs/index.md

```diff
@@ -4,8 +4,8 @@ nRF Connect Bluetooth® Low Energy helps you learn how to configure and test Blu
 
 ## Supported devices
 
-- PCA10056 nRF52840 Development Kit
-- PCA10040 nRF52 Development Kit
-- PCA10059 nRF52840 Dongle
-- PCA10028 nRF51 Development Kit
-- PCA10031 nRF51 Dongle
\ No newline at end of file
+- nRF52840 Development Kit (PCA10056)
+- nRF52 Development Kit (PCA10040)
```

Comment:
How about moving the nRF52 DK a line up or down? I think it would be nice to have the nRF52840 DK and nRF52840 Dongle next to each other. 

Maybe generally sort them alphabetically, so that the 51s are at the top and we have it like this?
```md
- nRF51 Development Kit (PCA10028)
- nRF51 Dongle (PCA10031)
- nRF52 Development Kit (PCA10040)
- nRF52840 Development Kit (PCA10056)
- nRF52840 Dongle (PCA10059)
```